resources/config.rb

```diff
@@ -17,14 +17,16 @@
 default_action :configure
 
 action :configure do
+  product = ChefUtils::Dist::Infra::PRODUCT || 'Configuration Manager'
+
```

Comment:
Have you tested this on Chef 15?

Reply:
Seems that the answer was no and i didn't want to invest in fixing this so i changed the message to be generic and removed the product notation all together.

Reply:
Interesting I would be curious what the error was, I would think its probably a nomethod error and we could easily rescue that.

Reply:
yeah something like a no method error. tried a quick rescue and i think i got it wrong because it just output blank instead. I had already gone over my $dayjob allocated time for getting this out and 1.0.0 released so we can fix chef 16 and the template change so i sadly didn't have more time to invest in it. Atleast i managed to get the documentation update done also which was nice :) 

maybe ill find some time to fiddle with this and cut a PR to fix the naming when i get a free moment to myself

Reply:
Completely understandable

> I had already gone over my $dayjob allocated time for getting this out and 1.0.0 released so we can fix chef 16 and the template change so i sadly didn't have more time to invest in it. 

I know this only too well. We are actually looking moving our atlantis to k8s next quarter so its actually very opportune timing to hand this off to the community. I will absolutely still try to stick around for reviews where I can.